web/cypress/e2e/group_1_onboarding/accessibility.spec.ts

```diff
@@ -0,0 +1,86 @@
+import "cypress-axe";
+import { setMobileViewport } from "../../support/helpers/helpers";
+import { onOnboardingPageStartingBusiness } from "../../support/page_objects/onboardingPageNew";
+
+describe("Automated Accessibilty Testing [feature] [all] [group1]", () => {
+  describe("Onboarding Desktop Accessibility Test", () => {
+    beforeEach(() => {
+      cy.loginByCognitoApi();
+      cy.injectAxe();
+    });
+
+    it("Accessibility for All Other Businesses", () => {
```

Comment:
This test and the mobile test are virtually identical. Cypress gives us the ability to loop over a single test with multiple scenarios. This should give us the ability to toggle between the desktop and mobile viewport.

Not a hard requirement for getting this work in now, but if we plan to expand on this file, it might be a good framework for us to have in place so that we can easily add 1 test and check mobile and desktop.